app/metrics.yaml

```diff
@@ -8,6 +8,26 @@ no_lint:
   - CATEGORY_GENERIC
```

Comment:
nit: can we squash the commits?

Also, the extended commit summary isn't super helpful:

>remove logs
>
>fix formatting
>
>fix formatting

Can we remove that too?

Reply:
sure!